astroquery/utils/tap/model/tests/test_job.py

```diff
@@ -26,13 +26,13 @@ def data_path(filename):
     data_dir = os.path.join(os.path.dirname(__file__), 'data')
     return os.path.join(data_dir, filename)
 
-class TestJob(unittest.TestCase):
 
+class TestJob(unittest.TestCase):
 
     def test_job_basic(self):
         job = Job(async=False)
         res = job.is_sync()
-        assert res == True, \
```

Comment:
Is there any chance of this causing a problem?  i.e., is there any chance that `res==2`, which would now evaluate to `True` but previously would not?